vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.h

```diff
@@ -437,6 +437,16 @@ class LedgerImpl : public ledger::Ledger,
   scoped_refptr<base::SequencedTaskRunner> GetTaskRunner();
   void GetRewardsInternalsInfo(ledger::RewardsInternalsInfo* info) override;
 
+  void RemoveData(int32_t remove_mask,
+      ledger::OnDataRemovedCallback callback) override;
+
```

Comment:
nit: why all these extra spaces?

Reply:
It seems most of the declarations are crlf separated.